docs/docs/README.md

```diff
@@ -11,28 +11,37 @@ meta:
 
 To invite the bot, you must use [this link](https://discordapp.com/oauth2/authorize?client_id=474051954998509571&scope=bot&permissions=93248).
 
-Here you can choose the server you wish to invite the bot to and choose what permissions it should have. It is best to leave the permissions alone or you'll most likely run into issues. The bot is only assigned the permissions it needs. Those permissions are:
+Here you can choose the server you wish to invite the bot to and choose what permissions it should have. It's best to leave the permissions alone or you'll most likely run into issues. The bot is only assigned the permissions it needs. Those permissions are:
 
-* Read Messages & View Channels (`VIEW_CHANNEL`)
+* View Channels (`VIEW_CHANNEL`)
 * Send Messages (`SEND_MESSAGES`)
 * Manage Messages (`MANAGE_MESSAGES`)
 * Embed Links (`EMBED_LINKS`\)
 * Read Message History (`READ_MESSAGE_HISTORY`)
 * Add Reactions (`ADD_REACTIONS`)
 
-## Set Up the Bot
+## Setting Up the Bot
 
-Once the bot is in your server, there are a few housekeeping things you need to do (you must have the Manage Server (`MANAGE_GUILD`) permission to complete these steps).
+Once the bot is in your server, there are a few housekeeping things you need to do. (You must have the Manage Server (`MANAGE_GUILD`) permission to complete these steps.)
 
-Also by default, the prefix is`,`and the bot searches for `#suggestions` as the default user suggestions channel. The bot mention (`@Suggestions#2602`) can be used as a prefix for commands as well if you ever incorrectly set the prefix or forget the prefix in general.
+Also by default, the prefix is `,` and the bot searches for `#suggestions` as the default suggestions channel. The bot mention (`@Suggestions#2602`) can be used as a prefix for commands as well if you ever incorrectly set the prefix or forget the prefix in general.
```

Comment:
``and the bot searches for `#suggestions` as the default suggestions channel.`` This is no longer the default behavior as far as I am aware. We should now point users to use their own channel which they must configure.

Reply:
The bot will only search for a channel called `#suggestions` when using the `suggest` command. If the channel doesn't exist when the user is running a command, the error will display prompting the channel to be set. Otherwise, the channel will be set before the rest of the `suggest` command is executed. So technically, this message here is still correct. 

I originally removed it but re-added it due to the confusion and issues it was causing. I removed it in the first place unknowing because I didn't know that the `messageReactionAdd` event was broken (https://github.com/suggestionsbot/suggestions-bot/commit/bb0decf96fb899c97d7c8eb59c6a52cff6139df4)